### DISCUSSION TITLE
How to insert into with single struct and multiple struct?

### DISCUSSION THREAD

**Discussion by marcopolooo** [jackc/pgx#1988] [Q&A]
I would like to know does pgx supports "insert into" with a single struct or multiple structs, perhaps we can use an array or slice struct? Thank you in advance

**Comment by jackc** [comment]
pgx is a database driver, not an ORM. It does not have any builtin functionality to insert entire structures. With pgx you would use the `Conn.Exec` method with the SQL that performs your desired operation. If you are looking for ORM functionality then see if one of the libraries [linked in the README](https://github.com/jackc/pgx?tab=readme-ov-file#3rd-party-libraries-with-pgx-support) will do what you need. And if you are using the database/sql interface, then gorm seems to be a popular choice.
